.github/workflows/keepalive.yml

```diff
@@ -0,0 +1,14 @@
+name: Keepalive
+
+on:
+  schedule:
```

Comment:
```suggestion
  # Run once a month at a semi-random time
  schedule:
```

Reply:
erk, the cron is wrong, running once per month, should be once per week

Reply:
Have added a composer package that converts crons to human readable, am using that instead